src/lib/serviceRoutines/postQueryContext.cpp

```diff
@@ -150,7 +150,7 @@ static bool queryForward(ConnectionInfo* ciP, QueryContextRequest* qcrP, QueryCo
     return false;
   }
 
-  LM_T(LmtCPrForwardRequestPayload, ("forward queryContext response payload: %s", out.c_str()));
+  LM_T(LmtCPrForwardResponsePayload, ("forward queryContext response payload: %s", out.c_str()));
```

Comment:
Good catch... currently it seems that LmtCPrForwardResponsePayload is not used in any place (https://github.com/telefonicaid/fiware-orion/search?q=LmtCPrForwardResponsePayload&unscoped_q=LmtCPrForwardResponsePayload)

Could you do the same fix for postUpdateContext.cpp file, pls?

Reply:
sure

Reply:
Fixed in e354378